libraries/apollo-api/api/apollo-api.api

```diff
@@ -989,6 +989,10 @@ public final class com/apollographql/apollo3/api/http/internal/UrlEncodeKt {
 	public static final fun urlEncode (Ljava/lang/String;)Ljava/lang/String;
 }
 
+public final class com/apollographql/apollo3/api/internal/ResponseParserKt {
+	public static final fun readErrors (Lcom/apollographql/apollo3/api/json/JsonReader;)Ljava/util/List;
```

Comment:
Should it be `@ApolloInternal`?

Reply:
I thought it could be useful in general but you're right, easier to remove the `@ApolloInternal` later if needed than add it.